CHANGELOG.md

```diff
@@ -2,6 +2,33 @@
 
 ## master
 
+## 0.8.0 (2018-10-01)
```

Comment:
Let's leave the change log entry under `master` section. We don't know the date and version of the release  this feature will be included)